modules/rostests/winetests/kernel32/path.c

```diff
@@ -1935,11 +1935,15 @@ static void test_SearchPathW(void)
     static const WCHAR extW[] = {'.','e','x','t',0};
     static const WCHAR dllW[] = {'.','d','l','l',0};
     static const WCHAR fileW[] = { 0 };
+    static const WCHAR* longDirName =  L"C:\\Program Files\\Very_long_test_path_which_can_trigger_heap_overflow_test_1234567890______________________________________________________AB";
```

Comment:
Are these tests _actually_ present in wine? If they are not, please add your own tests in the corresponding module in "apitests" instead (and not "winetests").

And since you're interested in RtlDosSearchPath_Ustr, any custom extra tests should go into `apitests/ntdll/RtlDosSearchPath_Ustr.c`

Reply:
Ok. I will move this test to the apitest.

Reply:
Please note that you're also free to add extra tests for kernel32!SearchPathW function in a `SearchPath.c` file inside `apitests/kernel32`, as you see fit.